pymc3/tests/test_smc.py

```diff
@@ -13,15 +14,13 @@
 @pytest.mark.xfail(condition=(theano.config.floatX == "float32"), reason="Fails on float32")
 class TestSMC(SeededTest):
 
-    def setup_method(self):
-        super(TestSMC, self).setup_method()
+    def setup_class(self):
```

Comment:
just want to make sure this is intentional -- you'll now reuse the same test folder for every test in the class